integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -15,10 +15,15 @@ use crate::test_loop::env::TestLoopEnv;
 use crate::test_loop::utils::ONE_NEAR;
 
 /// Stub for checking Resharding V3.
-/// After uncommenting panics with
-/// StorageInconsistentState("Failed to find root node ... in memtrie")
+/// TODO(#11881): add the following scenarios:
+/// - Shard ids should not be contiguous. For now we reuse existing shard id
+/// which is incorrect!!!
+/// - Nodes must not track all shards. State sync must succeed.
+/// - Set up chunk validator-only nodes. State witness must pass validation.
+/// - Tx load must be consistent. Txs and receipts must cross resharding
```

Comment:
It might be better if test cross shard receipts in a separate test, but yes tx should be consistent